cypress/integration/plugins/query-workbench-dashboards/ui.spec.js

```diff
@@ -91,7 +91,7 @@ describe('Test PPL UI', () => {
 
   it('Test full screen view', () => {
     cy.get('.euiButton__text').contains('Full screen view').should('not.exist');
-    cy.get('.euiTitle').contains('Query Workbench').should('exist');
+    // cy.get('.euiTitle').contains('Query Workbench').should('exist');
```

Comment:
please remove the commented code if not needed anymore

Reply:
removed